crates/router/src/connector/paybox.rs

```diff
@@ -136,13 +132,24 @@ impl ConnectorCommon for Paybox {
 }
```

Comment:
remove build_error_response if not being used

Reply:
I think it is better to keep as it is used in all other flows